ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -80,29 +68,18 @@ type Props = ReturnType<typeof mapStateToProps> &
   ReduxMergedProps &
   OwnProps;
 
-const formatTextRecipient = (e: EnteBeneficiario): string => {
-  const denomUnitOper = fromNullable(e.denomUnitOperBeneficiario)
-    .map(d => ` - ${d}`)
-    .getOrElse("");
-  const address = fromNullable(e.indirizzoBeneficiario).getOrElse("");
-  const civicNumber = fromNullable(e.civicoBeneficiario)
-    .map(c => ` n. ${c}`)
-    .getOrElse("");
-  const cap = fromNullable(e.capBeneficiario)
-    .map(c => `${c} `)
-    .getOrElse("");
-  const city = fromNullable(e.localitaBeneficiario)
-    .map(l => `${l} `)
-    .getOrElse("");
-  const province = fromNullable(e.provinciaBeneficiario)
-    .map(p => `(${p})`)
-    .getOrElse("");
-
-  return `${e.denominazioneBeneficiario}${denomUnitOper}\n
-${address}${civicNumber}\n
-${cap}${city}${province}`;
-};
+const styles = StyleSheet.create({
+  lighterGray: { color: "#C1C9D2" },
```

Comment:
Since we use this color multiple times, could it be better to place it in **variables**?

Reply:
updated within https://github.com/pagopa/io-app/pull/1628/commits/28bd2d3ba35bb2780691fbc8071734711ef11507